### ISSUE TITLE
Cannot find parameter - Error binding null parameter SQLAlchemy

### ISSUE THREAD

**Issue by Apoalex92** [koxudaxi/local-data-api#49]
Hello,
I've encountered a problem when trying to call the the Execution API.

I'm performing a query like the one reported in the following, so I would like to extract all the employees belonging to a nation (given in the form of UUID) and with a specific language, if provided in input parameters:
`select * from employes emp where emp.nation = cast(:nation as UUID) and :lang is null or t.language = ( select id from languages where language = :lang);`

Given the parameters type in the form of:
```
	"parameters": [{
		"name": "lang",
		"value":  {
			"isNull": "true"
		}
	}, {
		"name": "nation",
		"value": {
			"stringValue": "9a370a89-b773-45d2-a0cd-9ffacefd54c8"
		}
	}]
```

I obtain the following error: _Cannot find parameter: lang_
Debugging the code i realized that the problem is related to _create_query_ method in _resource.py_ file.
It seems that the command _str(text_sql.bindparams(**params).compile(**kwargs))_ cannot bind null parameters.

I have temporarily fixed with the following code:

```
params_none = dict((k,v) for k,v in params.items() if v is None)
for key in params_none.keys():
    sql = sql.replace(f":{key}", "null")
```
before the _text_sql: TextClause = text(sql)_ statement.

If it could be useful, I'm using a PostgreSql database

**Comment by mreed19** [comment]
Just ran into this issue as well and the solution you gave fixed it for me!

**Comment by koxudaxi** [comment]
@Apoalex92 
Thank you for creating the issue.

I'm happy to get a great workaround :+1: 
I will fix it this weekend.

**Comment by koxudaxi** [comment]
@Apoalex92 
@mreed19

I have released a fixed version as  `0.4.8` :tada:

**Comment by Apoalex92** [comment]
@koxudaxi Thank you so much!
